### PR TITLE
testing/cpputest: new aport

### PR DESCRIPTION
**Pull request by shiva**
[](url)    http://cpputest.github.io/
    CppUTest - unit testing and mocking framework for C/C++

### CODE REVIEW

**Comment by shiva** [comment]
What can I do to get this reviewed?

File: testing/cpputest/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Contributor: Shiva Velmurugan <shiv@shiv.me>
+# Maintainer: Shiva Velmurugan <shiv@shiv.me>
+pkgname=cpputest
+pkgver=3.8
+pkgrel=1
```
Comment (jirutka) [inline testing/cpputest/APKBUILD]: The initial pkgrel should be 0.

File: testing/cpputest/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+subpackages="$pkgname-doc"
+source="https://github.com/cpputest/${pkgname}/releases/download/v${pkgver}/${pkgname}-${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver-build/"
+
+build() {
+    mkdir -p "$builddir" && \
+    cd "$builddir" &&\
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr $srcdir/$pkgname-$pkgver
+    make || return 1
+}
```
Comment (jirutka) [inline testing/cpputest/APKBUILD]: Please use only tabs for indentation (and alignment) in APKBUILDs and other shell scripts, not spaces.

File: testing/cpputest/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+makedepends="cmake"
+install=""
+subpackages="$pkgname-doc"
+source="https://github.com/cpputest/${pkgname}/releases/download/v${pkgver}/${pkgname}-${pkgver}.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver-build/"
+
+build() {
+    mkdir -p "$builddir" && \
+    cd "$builddir" &&\
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr $srcdir/$pkgname-$pkgver
```
Comment (jirutka) [inline testing/cpputest/APKBUILD]: Variables with file paths which are set outside of the abuild should be always quoted (`"$srcdir"/$pkgname-$pkgver`).

File: testing/cpputest/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    make || return 1
+}
+
+package() {
+    cd "$builddir"
+    make DESTDIR="$pkgdir" PREFIX="/usr" install || return 1
+    install -Dm644 "${srcdir}/${pkgname}-${pkgver}/COPYING" "${pkgdir}/usr/share/licenses/$pkgname/COPYING"
```
Comment (jirutka) [inline testing/cpputest/APKBUILD]: Please wrap lines to 80 (max 100) characters.

**Comment by shiva** [comment]
@jirutka Thanks a lot for your comments. Will get back with changes.

File: testing/cpputest/APKBUILD (hunk 1 above)
Comment (shiva) [inline testing/cpputest/APKBUILD]: done

File: testing/cpputest/APKBUILD (hunk 3 above)
Comment (shiva) [inline testing/cpputest/APKBUILD]: done

File: testing/cpputest/APKBUILD (hunk 2 above)
Comment (shiva) [inline testing/cpputest/APKBUILD]: done

File: testing/cpputest/APKBUILD (hunk 4 above)
Comment (shiva) [inline testing/cpputest/APKBUILD]: done

**Pull request closed without merging**